src/server/cluster/cluster_slot_migration.cc

```diff
@@ -101,6 +103,21 @@ ClusterSlotMigration::Info ClusterSlotMigration::GetInfo() const {
   return {ctx.host, ctx.port, state_};
 }
 
+bool ClusterSlotMigration::TrySetStableSync(uint32_t flow) {
+  DCHECK(shard_flows_.size() > flow);
+  shard_flows_[flow]->SetStableSync();
+  auto res = std::all_of(shard_flows_.begin(), shard_flows_.end(),
+                         [](const auto& el) { return el->IsStableSync(); });
+  if (res) {
+    // TODO make this when we set new config
+    state_ = ClusterSlotMigration::C_STABLE_SYNC;
+    for (auto& flow : shard_flows_) {
+      flow->Cancel();
```

Comment:
why do you cancle?

Reply:
to stop the process because we don't do it in config,  I will remove it when add this functionality

Reply:
Add a `// TODO` then?